examples/erc20-react-example/src/reportWebVitals.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```

Comment:
please don't use `eslint-disable`

Reply:
this is autogenerate file from crap, can we just remove this file?

Reply:
In this project there is no point in using [`web-vitals`](https://web.dev/vitals/#overview), it can be removed but be careful.

btw it took me literally 30 sec to fix all lint issues on it